src/Explorer/Graph/GraphExplorerComponent/ArraysByKeyCache.ts

```diff
@@ -77,8 +79,10 @@ export class ArraysByKeyCache<T> {
   private reduceCacheSize(): void {
     // remove an key and its array
     const oldKey = this.keyQueue.shift();
-    this.totalElements -= this.cache[oldKey].length;
-    delete this.cache[oldKey];
+    if (oldKey) {
```

Comment:
can you check if this also fixes the compiler error

  if (this.keyQueue.length > 0)

Reply:
![image](https://user-images.githubusercontent.com/64865559/104082653-87b08380-51fd-11eb-92ff-1b2e56701596.png)
thanks, Armando, I will not changed